### PR TITLE
DROOLS-1463 DMN error message consolidation

### PR DESCRIPTION
**Pull request by tarilabs**
raised just for draft / will keep updated

### CODE REVIEW

File: kie-dmn-core/src/main/java/org/kie/dmn/core/util/Msg.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.kie.dmn.core.util;
+
+
+public class Msg extends org.kie.dmn.feel.util.Msg {
```
Comment (tarilabs) [inline kie-dmn-core/src/main/java/org/kie/dmn/core/util/Msg.java]: not 100% sure if to extend or create a "duplicated utility" class here

Comment (etirelli) [inline kie-dmn-core/src/main/java/org/kie/dmn/core/util/Msg.java]: I think we don't need to extend it. Just create a standalone class for dmn-core.

**Comment by etirelli** [comment]
@tarilabs the idea is correct, we will need to figure out how to deal with the message type code, but other than that, looks good.

**Comment by tarilabs** [comment]
the commit just added should cover all message String, for later i18n purposes.
I will double check coverage is good and complete the rest of the task in a subsequent commit later.

**Comment by etirelli** [comment]
Looks good @tarilabs . I am not sure I would have created a separate MessageTypeImpl class, but it seems fine and we can always change it later. 

+1 to merge.

**Pull request merged**